etsin_finder/frontend/js/components/qvain/files/fileSelector.jsx

```diff
@@ -28,97 +29,84 @@ export class FileSelectorBase extends Component {
     }
   }
 
+  drawFile = (f) => {
+    const {
+      toggleSelectedFile,
+      setMetadataModalFile
+    } = this.props.Stores.Qvain
+
+    return (
+      <li key={f.identifier} style={{ paddingLeft: '20px', display: 'flex', alignItems: 'center' }}>
```

Comment:
I do not recommend using inline styles and we should refactor out all uses in the codebase at some point, however it is ok now and you don't need to change it.
https://reactjs.org/docs/faq-styling.html#are-inline-styles-bad